inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/gather/gather_kernel_ref.cpp

```diff
@@ -145,7 +188,16 @@ JitConstants GatherKernelRef::GetJitConstants(const gather_params& params) const
     jit.AddConstant(MakeJitConstant("INDICES_INDEX_ORDER", GetIndecesIdxOrder(params, GetGatherChannelIndex(params))));
 
     if (!params.fused_ops.empty()) {
-        FusedOpsConfiguration conf = { "", {"b", "f", "y", "x"}, "val", params.inputs[0].GetDType() };
+        std::vector<std::string> idx_order;
+        if (params.inputs[0].GetDims().size() <= 4) {
+            idx_order = {"b", "f", "y", "x"};
+        } else if (params.inputs[0].GetDims().size() == 5) {
+            idx_order = {"b", "f", "z", "y", "x"};
+        } else if (params.inputs[0].GetDims().size() == 6) {
+            idx_order = {"b", "f", "w", "z", "y", "x"};
+        }
```

Comment:
Can we use `std::vector<std::string> GetOrder(size_t size)` function here?

Reply:
Done